src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
+}
+
+.logo {
```

Comment:
Consider using BEM naming conventions for the header and logo classes. For example `header__logo` for the logo element.